### DISCUSSION TITLE
Pasting an enormous amount of text into the new Find UI causes the browser to hang briefly

### DISCUSSION THREAD

**Discussion by j-f1** [community/community#40359] [Repositories]
### Select Topic Area

Bug

### Body

I had previously copied a medium sized JS file (752kB) into my clipboard to format it. Then, I thought I’d selected a token from that file to search for on GitHub. When I pasted, I discovered that I had in fact not actually copied that token. So I ended up pasting 752kB of text into the Find panel on the right sidebar of the code viewer. It told me that the search had to be less than 1,000 characters, and when I tried to delete the text and try again, the page froze for a bit. Would it be possible to set a reasonable upper bound on the input field via the `maxlength` HTML property so that long text can be automatically truncated?

**Comment by ossanna16** [comment]
Hi there @j-f1 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
